docs/shared/recipes/plugins/create-package.md

```diff
@@ -0,0 +1,75 @@
+# Creating a create package
+
+When a package named create-{x} is published to npm, it can be invoked via any of the following:
+
+- npx create-{x}
+- yarn create {x}
+- npm init {x}
+- pnpm init {x}
+
+These packages are used to set up a new project in some form. As an example, consider `create-nx-plugin`, `create-nx-workspace`, or perhaps `create-react-app`.
+
+There are a few paths to create a package that will work with `create-nx-workspace`'s public API to setup a new workspace that uses your Nx plugin:
+
+- When creating a new nx plugin via `create-nx-plugin` to setup a workspace, provide a name when prompted for your create package
```

Comment:
- Create a new workspace for your plugin with `create-nx-plugin` and provide a name when prompted for your create package